package.json

```diff
@@ -122,6 +124,7 @@
     "react-native-document-picker": "^8.0.0",
     "react-native-fast-image": "^8.6.3",
     "react-native-fs": "^2.20.0",
+    "react-native-geolocation": "^1.0.0",
```

Comment:
do we need both of these two packages?

- "@react-native-community/geolocation"
- "react-native-geolocation"